### DISCUSSION TITLE
تغییر شیوه جمع‌آوری لینک‌ها

### DISCUSSION THREAD

**Discussion by Hameds** [softwaretalks/newsletter#355] [Ideas]
<div dir='rtl'>

پیرو نظری که در https://github.com/softwaretalks/newsletter/issues/346 عرض کردم، ایده کلی این هست که تغییراتی در شیوه گردآوری لینک‌ها داشته باشیم. من برای خبرنامه دیگری در حال برنامه‌ریزی هستم و برای softwaretalks دو پیشنهاد زیر رو دارم:

۱- استفاده از یک افزونه مرورگر که کمک کنه هر صفحه/مقاله‌/ویدئویی که مناسب هست در فرمت issueهای محتوایی به مخزن اضافه بشه. کاربر در تنظیمات نامش رو وارد می‌کنه، عنوان و لینک به صورت اتوماتیک وارد می‌شه. عنوان و توضیحات قابل ویرایش هستند.

۲- استفاده از یک سرویس خبرخوان که فید وبلاگ‌ها و منابعی که محتوا دارند رو به اون اضافه کنیم و از بین مطالبی که منتشر می‌شه تعدادی رو برای خبرنامه انتخاب کنیم.

دو پیشنهاد بالا رو می‌شه در سطحی براشون کد نوشت تا اتوماتیک‌تر باشند. حتی پیشنهاد دوم رو می‌شه با اولی ترکیب کرد و مثلاً از بین مطالب خبرخوان (RSS Reader) با کمک افزونه تعدادی رو کاندیدای انتشار در خبرنامه کرد.

برای منابع، می‌شه از سایت‌ها/وبلاگ‌هایی که تا حالا در خبرنامه‌های ارسالی فعلی ازشون مطلبی داشتیم استفاده کرد. اون‌هایی که RSS دارند کاندیدای مناسبی هستند و با توجه به اینکه یک بار [یا بیشتر] مطلبی ازشون در خبرنامه منتشر شده احتمالاً مستعد انتخاب برای انتشار مطالب جدیدتر هم هستند.

</div>

**Comment by ohmydevops** [comment]
<div dir='rtl'>

سلام حامد جان امیدوارم حالت خوب باشه

من الان این پیام رو خوندم و فکرم رو مشغول کرد. فکر کنم فردا صبح با ذهن بازتری میتونم بیشتر بهش فکر کنم و نظراتم رو بنویسم ولی ایده خیلی جالبی رو مطرح کردی. فردا توضیحاتم رو میزارم اینجا

</div>

**Comment by ohmydevops** [comment]
<div dir='rtl'>

سلام شب بخیر
مباحثی که مطرح کردی به نظر میاد ایده اصلیش تسهیل‌گری مشارکت کاربرانه تا بتونیم فرایند رو برای کاربران مشارکت‌کننده،‌ راحت‌تر کنیم. 

من با همه موارد موافقم. مورد اول که گفتی کار کاربر رو راحت میکنه، مورد دوم کار خودمون رو. ولی خب، هدف صرفا بودن محتوا در هر شماره نیست، هدف، بودن محتوای باکیفیت گلچین شده‌است. البته این ایده‌آلمونه طبیعتا و این موارد ذکر شده بهش کمک میکنه.

میتونم برنامه‌ریزی افزونه مرورگر رو کنار کارهای دیگه خبرنامه شروع کنم. و البته از دوستان متخصص‌تر هم کمک بگیریم. چون من تجربه‌ام برای نوشتن افزونه‌ها در حد هلوورلد هستش

</div>

**Comment by ohmydevops** [comment]
من متوجه این بخش نشدم؟ 

```
من برای خبرنامه دیگری در حال برنامه‌ریزی هستم 
```

منظور جایگزین برای همین خبرنامه هست یا موضوع شخصی دیگری؟

**Reply by Hameds** [reply]
<div dir='rtl'>

منظورم برای خبرنامه «کارِج» بود. برای تسهیل جمع‌آوری اطلاعات اونجا برنامه‌ریزی می‌کردم و گفتم همون کار رو البته با رعایت ملاحظات محتوایی برای این خبرنامه هم اجرایی کنیم.

</div>
